.github/ISSUE_TEMPLATE/bug_short_name.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug_short_name
```

Comment:
@MariaMaximova, what is the difference between `Bug report` and `Bug short name`?